### ISSUE TITLE
question about the long ranger basic

### ISSUE THREAD

**Issue by longzhangnation** [bcgsc/arcs#90]
hello,
  I was using the software to scaffold my genome assembly . But I meet some issues. I get several fastq file from sequencing center . But I don't know how to deal with them before starting. These files are as "`B1_S13_L003_R1_001.fastq.gz`,`B1_S13_L003_R2_001.fastq.gz`,`B1_S13_L003_I1_001.fastq.gz`,`B1_S13_L003_I2_001.fastq.gz`". I know the R1 and R2 are raw chromium sequencing data . But how do I process the data I1 and I2 ? Is it nesssary for me to process all of these data using `longranger basic` or  I just need to care about the R1 and R2 ?  Hope you can give me some advice.

Best

**Comment by longzhangnation** [comment]
In addition , these files are generated by bcf2fastq.

**Comment by lcoombe** [comment]
Hi @longzhangnation,

Yes, you need to process the reads using `longranger basic`. Take a look at their documentation here for guidelines on how to do that and the parameters required: https://support.10xgenomics.com/genome-exome/software/pipelines/latest/advanced/other-pipelines
After running `longranger basic`, you will have a single, interleaved fastq file with the barcodes in the read headers - this is your input to ARCS.

**Comment by longzhangnation** [comment]
OK. I will try read the document on the site . Thanks for you reply.

**Comment by lcoombe** [comment]
Great! Feel free to open another issue if you have any further questions.
